aea/cli/add.py

```diff
@@ -116,7 +116,7 @@ def add_item(ctx: Context, item_type: str, item_public_id: PublicId) -> None:
     is_local = ctx.config.get("is_local")
 
     ctx.clean_paths.append(dest_path)
-    if item_public_id in [DEFAULT_CONNECTION, DEFAULT_PROTOCOL, DEFAULT_SKILL]:
+    if item_public_id in [DEFAULT_CONNECTION, *LOCAL_PROTOCOLS, DEFAULT_SKILL]:
```

Comment:
We now have more than one local protocol. The previously "internal messages" are now also protocols.